solr/package.clj

```diff
@@ -17,15 +17,16 @@
 
 (io/make-parents "resources/.dummy")
 
-(doseq [[out-path schema stopwords]
-        [["resources/arabic.zip"    "solr/knowledge-expansion-ar.xml" "solr/lang/stopwords_ar.txt"]
-         ["resources/brazilian.zip" "solr/knowledge-expansion-br.xml" "solr/lang/stopwords_br.txt"]
-         ["resources/german.zip"    "solr/knowledge-expansion-de.xml" "solr/lang/stopwords_de.txt"]
-         ["resources/english.zip"   "solr/knowledge-expansion-en.xml" "solr/lang/stopwords_en.txt"]
-         ["resources/spanish.zip"   "solr/knowledge-expansion-es.xml" "solr/lang/stopwords_es.txt"]
-         ["resources/french.zip"    "solr/knowledge-expansion-fr.xml" "solr/lang/stopwords_fr.txt"]
-         ["resources/italian.zip"   "solr/knowledge-expansion-it.xml" "solr/lang/stopwords_it.txt"]
-         ["resources/japanese.zip"  "solr/knowledge-expansion-jp.xml" "solr/lang/stopwords_jp.txt"]]]
+(doseq [[out-path schema stopwords protwords]
+        [["resources/arabic.zip"    "solr/knowledge-expansion-ar.xml" "solr/lang/stopwords_ar.txt" "solr/lang/protwords_ar.txt"]
+         ["resources/brazilian.zip" "solr/knowledge-expansion-br.xml" "solr/lang/stopwords_br.txt" "solr/lang/protwords_br.txt"]
+         ["resources/german.zip"    "solr/knowledge-expansion-de.xml" "solr/lang/stopwords_de.txt" "solr/lang/protwords_de.txt"]
+         ["resources/english.zip"   "solr/knowledge-expansion-en.xml" "solr/lang/stopwords_en.txt" "solr/lang/protwords_en.txt"]
+         ["resources/spanish.zip"   "solr/knowledge-expansion-es.xml" "solr/lang/stopwords_es.txt" "solr/lang/protwords_es.txt"]
+         ["resources/french.zip"    "solr/knowledge-expansion-fr.xml" "solr/lang/stopwords_fr.txt" "solr/lang/protwords_fr.txt"]
+         ["resources/italian.zip"   "solr/knowledge-expansion-it.xml" "solr/lang/stopwords_it.txt" "solr/lang/protwords_it.txt"]
+         ["resources/japanese.zip"  "solr/knowledge-expansion-jp.xml" "solr/lang/stopwords_jp.txt" "solr/lang/protwords_jp.txt"]]]
```

Comment:
These lines are getting kind of long.  Mind splitting each over two lines? Eg:
>
["resources/arabic.zip"    "solr/knowledge-expansion-ar.xml"
"solr/lang/stopwords_ar.txt" "solr/lang/protwords_ar.txt"]